### PR TITLE
chore: fix wrappers' lib name

### PR DESCRIPTION
**Pull request by pcnc**
* Wrappers saves its lib `.so` files using the current version in its name - e.g. `wrappers-0.1.16.so`, which makes pg_upgrade crash whenever a version is upgraded
* post-upgrade PG loads wrappers just fine since it isn't keeping any equivalence of extname<>libfilename - just loads them all up, then searches for any registry entry

### CODE REVIEW

File: ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh [hunk 1]
```diff
@@ -247,6 +248,18 @@ function initiate_upgrade {
...
     cp --remove-destination "$PGLIBNEW"/*.control "$PGSHARENEW/extension/"
     cp --remove-destination "$PGLIBNEW"/*.sql "$PGSHARENEW/extension/"
+
+    if [ -d "$PGLIBOLD" ]; then
+        WRAPPERS_LIB_PATH=$(find "$PGLIBNEW" -name "wrappers*so")
+        if [ -f "$WRAPPERS_LIB_PATH" ]; then
+            OLD_WRAPPER_LIB_PATH=$(find "$PGLIBOLD" -name "wrappers*so")
+            if [ -f "$OLD_WRAPPER_LIB_PATH" ]; then
+                LIB_FILE_NAME=$(basename "$OLD_WRAPPER_LIB_PATH")
+                cp "$WRAPPERS_LIB_PATH" "$PGLIBNEW/${LIB_FILE_NAME}"
```
Comment (darora) [inline ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh:269]: ooof

File: ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh [hunk 2]
```diff
@@ -247,6 +248,18 @@ function initiate_upgrade {
 
     cp --remove-destination "$PGLIBNEW"/*.control "$PGSHARENEW/extension/"
     cp --remove-destination "$PGLIBNEW"/*.sql "$PGSHARENEW/extension/"
+
+    if [ -d "$PGLIBOLD" ]; then
+        WRAPPERS_LIB_PATH=$(find "$PGLIBNEW" -name "wrappers*so")
+        if [ -f "$WRAPPERS_LIB_PATH" ]; then
+            OLD_WRAPPER_LIB_PATH=$(find "$PGLIBOLD" -name "wrappers*so")
```
Comment (darora) [inline ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh]: the deb upgrades ensure that we won't match multiple files here? (not sure of behaviour of `-f` if multiple matches are passed in )

File: ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh (hunk 1 above)
Comment (darora) [reply ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh:269]: might be worth adding comments here on why we're doing this, and maybe also adding a few echo statements to enhance traceability

**Review by darora — APPROVED** [review]

File: ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh (hunk 2 above)
Comment (pcnc) [reply ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh]: Good point - limited output.

**Pull request merged**